### ISSUE TITLE
LearningRateMonitor KeyError with multiple parameter groups and no LR scheduler

### ISSUE THREAD

**Issue by eladsegal** [PyTorchLightning/pytorch-lightning#10024]
## 🐛 Bug
This bug is due to the change in https://github.com/PyTorchLightning/pytorch-lightning/pull/9786:
When there is an optimizer with multiple parameter groups and no learning rate scheduler,
`_find_names_from_optimizers` returns keys with suffix but then the suffix is added again in `_get_lr_momentum_stat`, resulting in a key error. 
It only happens for multiple parameter groups, as for a single group `_add_suffix` doesn't add the suffix.

**Comment by rohitgr7** [comment]
hey!
created a fix here: https://github.com/PyTorchLightning/pytorch-lightning/pull/10044
can anyone confirm if it's working for them now?
thanks :)
